_layouts/master.html

```diff
@@ -26,7 +26,7 @@
               <a class="nav-link" href="https://github.com/swaywm/sway/wiki">Wiki ≫</a>
             </li>
             <li class="nav-item">
-              <a class="nav-link" href="https://webchat.freenode.net/?channels=sway&uio=d4">Support Chat ≫</a>
+              <a class="nav-link" href="https://kiwiirc.com/nextclient/irc.libera.chat:+6697/#sway">Support Chat ≫</a>
```

Comment:
Hm, I'm not comfortable redirecting to kiwiirc.com, a third-party bridge. I'd rather wait for Libera Chat to gain an official web client.

I don't have many other ideas. Ideally we'd be using `ircs://irc.libera.chat/#sway`, but users without an IRC client installed will be confused.

Regardless, it would be a good idea to add `title="#sway on Libera Chat"`.

Reply:
Maybe split it to the irc link plus a webchat link, or create a landing page similar to matrix.to.

Another option would be to host your own instance of The Lounge or Kiwi IRC.

Reply:
I would say, there are different questions to resolve:

- How to inform users about possible ways to get help?
  - Currently, there is only one link to the "wrong" IRC channel. As far as I can tell, the channel is still available, but I don't know how many of the developers/supporters are still there.
  - There is also #sway-devel and it would be nice, if it can be found easier
  - Are there other (bridged) channels like Matrix or Telegram?
  - Any forums?
- Which webclient should be used?

Whatever decisions you take, I can update the PRs. Or you can reject them, if it is easier for you.

Reply:
The fact that we've moved from Freenode to Libera Chat doesn't really change anything else. In other words, users can get help on IRC, and we have no other official discussion medium.

I'd rather not mention #sway-devel in the READMEs, as it's not for end users. It's already mentioned in CONTRIBUTING.md.

There is no official Libera Chat web client yet, and I'm not comfortable recommending a third-party one, because any third party web client is basically a MITM. I don't think I'm ready to start hosting a public web client myself.

Maybe we should just link to ircs:// for now, and switch to the web client once Libera Chat have one.

Reply:
Maybe we should explicitly spell out the channel and network in the link text, e.g. "Support chat: #sway on Libera Chat".

Reply:
The matter is becoming [more pressing](https://mastodon.sdf.org/@kline/106299403921451814), so I've pushed an updated commit. Thanks!